Include rate limit headers in 429 exception

zmoon opened the pull request:
Resolves #382

zmoon committed:
Include rate limit headers in 429 exception

zmoon committed:
Include rate limit headers in 429 exception

russbiggs commented:
Thanks @zmoon, we're doing a big refactor of the codebase this week, so the base will change, once we get that merged in we can revisit and see how this matches up and/or needs any moving around to fit into the new structure.

caparker committed:
Update dependencies.py

No need to create a new exception, we can just rewrite the TOO_MANY_REQUESTS one to except headers.

caparker committed:
Update dependencies.py

No need to create a new exception, we can just rewrite the TOO_MANY_REQUESTS one to except headers.

caparker committed:
Update exceptions.py

Combined the two 'too many requests' methods

caparker committed:
Update exceptions.py

Combined the two 'too many requests' methods

russbiggs committed:
initial removal of v1 and v2 code (#381)

* initial removal of v1 and v2 code

* remove inner openaq_api directory

* initial poetry setup

* rename directory

* Adding poetry deployment

* Adding missing poetry

* Removing the global poetry install as test

* Another project file

* Moving cdk toml file to root

* Updated the cdk dependencies

* Updated imports and cleaned up env vars

* Removed duplicate vars

* Updating node and aws-cdk to be compatible

* Removed the waf_ip_block

  Causing duplication error

* Removed the api folder and combined project files

* Updated the asset location

* Cleaning up before merge

---------

Co-authored-by: Christian Parker <c.andrew.parker@gmail.com>

russbiggs committed:
auth email fix

russbiggs committed:
add error logging in auth routes

russbiggs committed:
return error for auth handlers

russbiggs committed:
smtp function fixes

russbiggs committed:
add additional logging for auth handlers

caparker committed:
Merge branch '429-ratelimit-headers' of github.com:zmoon/openaq-api-v2 into 429-ratelimit-headers

zmoon commented:
Should this be closed with #386 merged?
